### ISSUE TITLE
False positive

### ISSUE THREAD

**Issue by zerkms** [umbrae/jsonlintdotcom#30]
```json
{ "test": "\" }
```

this is reported as a valid JSON

**Comment by umbrae** [comment]
Jsonlint got stolen by a domain squatter and rewritten and they put ads on
it, so this repo is no longer accurate to what's on jsonlint.com :(
